js_modules/dagster-ui/packages/ui-core/src/runs/RunTimelineRowIcon.tsx

```diff
@@ -2,8 +2,9 @@ import {Icon} from '@dagster-io/ui-components';
 
 import {RowObjectType} from './RunTimelineTypes';
 import {assertUnreachable} from '../app/Util';
+import { TagIcon } from 'shared/graph/OpTags';
 
-export const RunTimelineRowIcon = ({type}: {type: RowObjectType}) => {
+export const RunTimelineRowIcon = ({type}: {type: RowObjectType | 'airflow'}) => {
```

Comment:
should we just add 'airflow' to RowObjectType?

Reply:
I tried it; but I think it ends up complicating the other places that it gets used.